bpe/train.py

```diff
@@ -74,57 +74,71 @@ def run(self):
 
 
 class ReturnnTrainBpeJob(Job):
-    def __init__(self, text_file, bpe_size, unk_label="UNK"):
+    def __init__(self, text_file, bpe_size, subword_nmt_repo=None, unk_label="UNK"):
         """
         Create Bpe codes and vocab files
         NOTE: This uses Albert's subword-nmt fork which is compatible to RETURNN BytePairEncoding class
```

Comment:
concrete reference to git-repo would be more helpful here, now that the repo is not pulled automatically anymore.